integration/integrate-with-image-vulnerability-scanners.adoc

```diff
@@ -35,12 +35,7 @@ You can set up {product-title-short} to obtain image vulnerability data from the
 [discrete]
 === Scanners included in {product-title-short}
 
-* Scanner V4 (Technology Preview): Beginning with {product-title-short} version 4.4, a new scanner is introduced that is built on link:https://github.com/quay/claircore[ClairCore], which also powers the link:https://github.com/quay/clair[Clair] scanner. Scanner V4 supports scanning of language and OS-specific image components. You do not have to create an integration to use this scanner, but you must enable it during or after installation. For version 4.4, if you enable this scanner, you must also enable the StackRox Scanner. For more information about Scanner V4, including links to the installation documentation, see xref:../operating/examine-images-for-vulnerabilities.adoc#about-scanner-v4_examine-images-for-vulnerabilities[About {product-title-short} Scanner V4].
-+
---
-:FeatureName: Scanner V4
-include::snippets/technology-preview.adoc[]
---
+* Scanner V4: Beginning with {product-title-short} version 4.4, a new scanner is introduced that is built on link:https://github.com/quay/claircore[ClairCore], which also powers the link:https://github.com/quay/clair[Clair] scanner. Scanner V4 supports scanning of language and OS-specific image components. You do not have to create an integration to use this scanner, but you must enable it during or after installation. For version 4.4, if you enable this scanner, you must also enable the StackRox Scanner. For more information about Scanner V4, including links to the installation documentation, see xref:../operating/examine-images-for-vulnerabilities.adoc#about-scanner-v4_examine-images-for-vulnerabilities[About {product-title-short} Scanner V4].
```

Comment:
From the first preview bullet, this instance of TP should be removed:

![image](https://github.com/openshift/openshift-docs/assets/87874774/db3865bb-daef-4076-8eaf-b05b0cf31d2f)